src/components/common/BasicMap.jsx

```
@@ -0,0 +1,139 @@
+import React, { useState, useCallback } from "react";
+import { Map } from "react-kakao-maps-sdk";
```

Biome (1.9.4)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)